content/post/kubestash-v2024.3.16/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Introducing KubeStash v2024.3.16
+date: "2024-03-19"
+weight: 10
+authors:
+- Md Anisur Rahman
+tags:
+- backup
+- cli
+- disaster-recovery
+- kubernetes
+- kubestash
+- restore
+---
+
+We are pleased to announce the release of `KubeStash v2024.3.16`, packed with new features and important bug fixes. You can check out the full changelog [HERE](https://github.com/kubestash/CHANGELOG/blob/master/releases/v2024.3.16/README.md). In this post, we'll highlight the key updates.
+
+### New Features
+
+1. We've added support for cross-namespace restore, giving you more flexibility when restoring your data ([#104](https://github.com/kubestash/apimachinery/pull/104)). With this update, you can now restore data to any namespace in your cluster. We've introduced a new field called `.spec.dataSource.namespace` within the RestoreSession (CRD). This field allows you to specify the namespace where the data source (repositories and snapshots) are located.
+
+    Here's an example of a `RestoreSession` with the `.spec.dataSource.namespace` field set:
+    ```yaml
+   apiVersion: core.kubestash.com/v1alpha1
+   kind: RestoreSession
+   metadata:
+      name: sample-mongodb-restore
+      namespace: <restore-session-namespace> 
```

Comment:
Give the actual namespace name here and explain in the description in which namespace the resources reside.